app/database_etl/airtable_records_handler/airtable_records_formatter.py

```diff
@@ -57,14 +57,14 @@ def standardize_airtable_data(df):
 
     # Convert elements that are "Not reported" or "Not Reported" or "NR" to None
     df.replace({'nr': None, 'NR': None, 'Not Reported': None, 'Not reported': None, 'Not available': None},
-                 inplace=True)
+               inplace=True)
 
     # Replace columns that should be floats with NaN from None and rescale to percentage
-    df[['ind_sp_n', 'ind_se_n']] = df[['ind_sp_n', 'ind_se_n']].replace({None: np.nan}) / 100
+    df[['ind_sp', 'ind_se']] = df[['ind_sp', 'ind_se']].replace({None: np.nan}) / 100
```

Comment:
why are these integers in airtable given that "sensitivity" and "specificity" are entered in as floats? seems confusing @rahularoradfs 

Would be cleaner to have both of them treated the same way by our code imo

Reply:
@rahularoradfs what do u think

Reply:
ind_sp_n and ind_se_n should be ints as they are counts - hence why we don't need to divide by 100 to put them on a percentage scale

ind_sp and ind_se are currently percentages expressed in range [0,100] that we are rescaling to [0,1] - so these should be floats